src/imio/esign/tests/test_browser_views.py

```
@@ -1,24 +1,246 @@
+class _BaseSessionViewTest(unittest.TestCase):
+    """Base test class with shared setUp for session view tests."""
+
+    layer = IMIO_ESIGN_INTEGRATION_TESTING
+
+    def setUp(self):
+        self.portal = self.layer["portal"]
+        self.request = self.layer["request"]
+        setRoles(self.portal, TEST_USER_ID, ["Manager"])
+
+        # Create user for signing
+        api.user.create(email="user1@sign.com", username="user1", password="password1")
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)